egs2/librimix/diar1/conf/tuning/train_diar_eda_5.yaml

```diff
@@ -0,0 +1,79 @@
+#    If "attractor" is "None" or not specified, SA-EEND will be used. 
```

Comment:
Can you try to simplify the config files to two settings (eend and eda, or only eda if eend is not ok for here)

Reply:
Simplified the descriptions in the config files.